lib/modulesync/settings.rb

```diff
@@ -12,8 +12,17 @@ def initialize(global_defaults, defaults, module_defaults, module_configs, addit
       @additional_settings = additional_settings
     end
 
+    def lookup_config(hash, filename)
+      fn = filename.chomp('.erb')
+      hash[fn] || hash["#{fn}.erb"] || {}
```

Comment:
The change I suggested doesn't change the logic, I'm not questioning that. I'm only suggesting that the code currently _removes_ the .erb extension and stores it as `fn`, then appends it again on this line. The `"#{fn}.erb"` code can simply use `filename`.